server/webhook.go

```diff
@@ -57,15 +58,63 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 	}
 
 	switch webhook.Event {
+	case zoom.EventTypeMeetingStarted:
+		p.handleMeetingStarted(w, r, b)
 	case zoom.EventTypeMeetingEnded:
 		p.handleMeetingEnded(w, r, b)
 	case zoom.EventTypeValidateWebhook:
 		p.handleValidateZoomWebhook(w, r, b)
+	case zoom.EventTypeRecordingCompleted:
+		p.handleRecordingCompleted(w, r, b)
+	case zoom.EventTypeTranscriptCompleted:
+		p.handleTranscriptCompleted(w, r, b)
 	default:
 		w.WriteHeader(http.StatusOK)
 	}
 }
 
+func (p *Plugin) handleMeetingStarted(w http.ResponseWriter, r *http.Request, body []byte) {
+	var webhook zoom.MeetingWebhook
+	if err := json.Unmarshal(body, &webhook); err != nil {
+		p.API.LogError("Error unmarshaling meeting webhook", "err", err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	meetingID, err := strconv.Atoi(webhook.Payload.Object.ID)
+	if err != nil {
+		p.API.LogError("Failed to get meeting ID", "err", err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	channelID, appErr := p.fetchChannelForMeeting(meetingID)
+	if appErr != nil {
+		http.Error(w, appErr.Error(), appErr.StatusCode)
```

Comment:
This is a bit of a weird pattern we have in this plugin that I think we should avoid. This http request that came in is from Zoom's webhook. I'm thinking we should always return a successful http status code, unless we find some incorrect with the wehook payload itself

Reply:
good point